Tools/peg_generator/scripts/benchmark.py

```diff
@@ -43,16 +47,17 @@
 
 def benchmark(func):
     def wrapper(*args):
+        print(f"{func.__name__}")
         times = list()
         for _ in range(3):
             start = time()
             result = func(*args)
             end = time()
             times.append(end - start)
-        memory = memory_profiler.memory_usage((func, args))
-        print(f"{func.__name__}")
         print(f"\tTime: {sum(times)/3:.3f} seconds on an average of 3 runs")
-        print(f"\tMemory: {max(memory)} MiB on an average of 3 runs")
+        if MEMORY_BENCHMARKS:
+            memory = memory_profiler.memory_usage((func, args))
+            print(f"\tMemory: {max(memory)} MiB on an average of 3 runs")
```

Comment:
Please add an `else:` that prints one line of instructions on how to install memory_profiler.